relay-cardinality/src/redis/cardinality.lua

```diff
@@ -86,20 +86,23 @@ local results = {
 --
 -- Returns the total amount of values added to the 'working set'.
 local function sadd(t, offset, max)
-    local added = 0;
+    local working_set_cardinality = 0;
```

Comment:
```suggestion
    local working_set_cardinality = 0
```

https://www.lua.org/pil/1.1.html